src/lib/dnssd/tests/TestTxtFields.cpp

```diff
@@ -983,23 +988,27 @@ void TxtFieldTcpSupport()
     strcpy(key, "T");
     strcpy(val, "1");
     FillNodeDataFromTxt(GetSpan(key), GetSpan(val), nodeData.resolutionData);
-    EXPECT_TRUE(nodeData.resolutionData.supportsTcp);
+    EXPECT_FALSE(nodeData.resolutionData.supportsTcpServer);
+    EXPECT_FALSE(nodeData.resolutionData.supportsTcpClient);
 
     // Test no other fields were populated
-    nodeData.resolutionData.supportsTcp = false;
+    nodeData.resolutionData.supportsTcpServer = false;
+    nodeData.resolutionData.supportsTcpClient = false;
     EXPECT_TRUE(NodeDataIsEmpty(nodeData));
 
     // False
     strcpy(key, "T");
     strcpy(val, "0");
```

Comment:
Again, seems like we need more coverage of different values here in the test.

Reply:
added bunch of more tests cases to cover all the values